modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="configuring_remote_write_storage_{context}"]
+= Configuring remote write storage
+
+You can configure remote write to enable Prometheus to send ingested metrics to remote systems for storage. 
+Doing so has no impact on how or for how long Prometheus stores metrics using local storage.
+
+.Prerequisites
+
+* *If you are configuring core OKD monitoring components:*
```

Comment:
Don't use OKD.

Reply:
Are there precedents we can follow? Searching the docs for `monitoring components`, I see instances of `*If you are configuring core {product-title} monitoring components*:`

Reply:
Thanks. Will update accordingly.

Reply:
Fixed